File: cocos2D/Lib/XFlatformCorePackage/WCP/src/WindowsCorePackage.WindowsPhone8.0/Source/Win8PhoneWindows8/ZOrdering.cs
namespace WCPToolkit
{
    namespace Utils
    {
        namespace ZOrdering
        {
            public enum ZPriority
            {
                BANNER = 0,
                INTERSTITIAL_NORMAL = 50,
                INTERSTITIAL_HIGH = 100
            }

            public sealed class ZOrder
            {
#if OS_W8
                public static void SetZPriority(Windows.UI.Xaml.UIElement elem, ZPriority priority)
                {
                    Windows.UI.Xaml.Controls.Canvas.SetZIndex(elem, (int)priority);
                }
#else
                public static void SetZPriority(System.Windows.UIElement elem, ZPriority priority)
                {
                    System.Windows.Controls.Canvas.SetZIndex(elem, (int)priority);
                }
#endif

            }
        }
    }
}

File: cocos2D/Lib/XFlatformCorePackage/WCP/src/WindowsCorePackage.Shared/Utils/ZOrdering.cpp
#include "pch.h"

#include "ZOrdering.h"

namespace WCPToolkit
{
	namespace Utils
	{
		namespace ZOrdering
		{
			void ZOrder::SetZPriority(Windows::UI::Xaml::UIElement^ elem, ZPriority priority)
			{
				Windows::UI::Xaml::Controls::Canvas::SetZIndex(elem, (int)priority);
			}
		}
	}
}
